package.json

```diff
@@ -33,6 +35,9 @@
     "@prisma/client": "^2.1.0",
     "casbin": "^5.0.4"
   },
+  "files": [
+    "lib"
```

Comment:
please append test、examples and prisma to files field.

Reply:
These files are useless to users

Reply:
@nodece

Reply:
It can help to users learn how use the prisma-adapter.